app/models/match.rb

```diff
@@ -22,6 +22,24 @@ class Match < ApplicationRecord
 
   validate :users_are_different
 
+  def self.create_and_notify(targets, user_id)
+    targets.find_each do |target|
+      created_match = create!(first_user_id: user_id, second_user_id: target.user_id,
+                              target_id: target.id)
+      notify_match(created_match)
+    end
+  end
+
+  def self.notify_match(match)
```

Comment:
Is it possible to not use a static method here? If so, you wouldn't need to pass match as a param, it will be available as self.

Reply:
Fixed in the previous PR.